frontend/taker.html

```diff
@@ -8,6 +8,6 @@
   </head>
   <body>
     <div id="root"></div>
-    <script type="module" src="../src/main_taker.tsx"></script>
+    <script type="module" src="./src/main_taker.tsx"></script>
```

Comment:
I think you could try and inline the content of `main_taker` here as well. Not sure if `rollup` is powerful enough for that but that would be kind of cool!

Then we don't need to have these `main_taker.tsx` and `main_maker.tsx` files ...